draft-ietf-quic-transport.md

```diff
@@ -1623,6 +1623,20 @@ also constrained in what they can send by the limits set by the congestion
 controller.  Clients are only constrained by the congestion controller.
 
 
+### Token Construction
+
+Tokens sent in NEW_TOKEN frames or Retry packets MUST be constructed in a way
+that allows the server to identity how it was provided to a client.  These
+tokens are carried in the same field, but require different handling from
+servers.
+
+A token MUST NOT include information that would allow values to be linked by an
```

Comment:
I think this requirement is specific to tokens used for validating future connections?

Therefore, something like "A token issued using NEW_TOKEN frames MUST NOT ..."

Reply:
Yeah, I moved too much up.